src/screens/AddCash/AddCash.js

```diff
@@ -62,48 +64,65 @@ const AddCash = () => {
   const fiatCurrency = useFiatCurrency();
   const colors = useThemeColors();
   const [currencySymbol, setCurrencySymbol] = React.useState(getCurrencySymbol(fiatCurrency));
+  const buttonDisable = value !== null && (Number(value) === 0 || !isValidFiatValue(value));
   const accounts = useRootSelector(accountsSelector);
   const activeAccount = getActiveAccount(accounts);
 
-  React.useState(() => {
-    Alert.alert(
-      t('servicesContent.wert.locationCheck.title'),
-      t('servicesContent.wert.locationCheck.message'), [
-        {
-          text: t('servicesContent.wert.locationCheck.no'),
-        },
-        {
-          text: t('servicesContent.wert.locationCheck.yes'),
-          onPress: () => {
-            setinUS(true);
-            setCurrencySymbol('$');
-          },
-        },
-      ]);
-  });
+  const residentSelected = (isUsResident: boolean) => {
+    if (isUsResident) {
+      setinUS(true);
+      setCurrencySymbol('$');
+    }
+  };
+
+  const networkSelected = () => {
+    if (inUS) {
+      openWert();
+    } else {
+      openRamp();
+    }
```

Comment:
Could some of these if statements also be single line, like so:
```suggestion
    if (inUS) openWert();
    else openRamp();
```